scripts/mining/kaggle/execution/vizminer.py

```diff
@@ -0,0 +1,375 @@
+import ast
```

Comment:
Let's move this file, and `mpl_seaborn_mining` and `plotly_mining` files into a single directory, say `viz_mining`. The contents of `mpl_seaborn_mining` and `plotly_mining` can be merged into this directory, where the miners can be placed in `mpl_seaborn_miner.py` and `plotly_miner.py`.